x-pack/plugins/aiops/public/components/time_field_warning.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiSpacer, EuiCallOut } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+
+export const TimeFieldWarning = () => {
```

Comment:
Nit - the text in this component is specific to pattern analysis. Should it be renamed to e.g. `PatternAnalysisTimeFieldWarning`?

Reply:
The component is now used for both pattern analysis and log rate analysis, I updated the text to a more generic `The analysis can only be run on data views with a time field.`. Updated the `i18n` ids too to remove the pattern analysis reference. 1e82368